lib/beaker/options/presets.rb

```diff
@@ -110,7 +110,7 @@ def env_vars
       def presets
         h = Beaker::Options::OptionsHash.new
         h.merge({
-          :project                => 'Beaker',
+          :project                => ENV['JOB_NAME'] || 'Beaker',
```

Comment:
Yep... this particular block is setting defaults, the block you referenced then overrides the defaults with the 'BEAKER_' ENV params.